### DISCUSSION TITLE
Can Inline Hints Toggle value remain the same after changing the theme?

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#7478] [Feature Requests]
I've seen that changing the theme also removes the Inline Hints (showing type information, for example). I think most users would want to keep seeing the Inlay Hints, just with different colors, instead of having to turn it on over and over while trying different themes.

**Comment by ghost** [comment]
Also, I see that Inline Hints appear as if the user had typed it, and it's not easy to visually identify when it's an inline hint Vs an actual type signature written by the user. Some other tools show the hint in a slightly different background color, a slightly smaller font size, like the following picture from Jetbrains RustRover:

![image](https://github.com/zed-industries/zed/assets/1676245/c31ac742-8be0-4efb-acd3-a600e87ca193)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
